schedule.html

```diff
@@ -0,0 +1,322 @@
+            </th>
+            <td>
+              WebVR intro: implementation status, obstacles, future plans
+            </td>
+          </tr>
+          <tr>
+            <th class="break">
+              09:45–10:00
+            </th>
+            <td>
+              Break
+            </td>
+          </tr>
+          <tr>
+            <th>
+              10:00–11:00
+            </th>
+            <td>
+              VR user interactions in browsers
+            </td>
+          </tr>
+          <tr>
+            <th>
+              11:00–12:00
+            </th>
+            <td>
+              Accessibility of VR experiences
+            </td>
+          </tr>
+          <tr>
+            <th class="break">
+              12:00–13:00
+            </th>
+            <td>
+              Lunch
+            </td>
+          </tr>
+          <tr>
+            <th>
+              13:00–14:00
+            </th>
+            <td>
+              Shared VR experiences
+            </td>
+          </tr>
+          <tr>
+            <th>
+              14:00–14:45
+            </th>
+            <td>
+              (to be announced)
+            </td>
+          </tr>
+          <tr>
+            <th class="break">
+              14:45–15:00
+            </th>
+            <td>
+              Break
+            </td>
+          </tr>
+          <tr>
+            <th>
+              15:00–16:00
+            </th>
+            <td>
+              Authoring VR experiences for the Web
+            </td>
+          </tr>
+          <tr>
+            <th>
+              16:00–17:00
+            </th>
+            <td>
+              (to be announced)
+            </td>
+          </tr>
+        </tbody>
+      </table>
+      <h4 id="day2">
+        Day 2: 20 October
+      </h4>
+      <table class="schedule">
+        <tbody>
+          <tr>
+            <th>
+              08:00–08:30
+            </th>
+            <td>
+              Registration
+            </td>
+          </tr>
+          <tr>
+            <th>
+              09:00–09:45
+            </th>
+            <td>
+              Streaming 360 video
+            </td>
+          </tr>
+          <tr>
+            <th class="break">
+              09:45–10:00
+            </th>
+            <td>
+              Break
+            </td>
+          </tr>
+          <tr>
+            <th>
+              10:00–11:00
+            </th>
+            <td>
+              360 video in HTML
+            </td>
+          </tr>
+          <tr>
+            <th>
+              11:00–12:00
+            </th>
+            <td>
+              (to be announced)
+            </td>
+          </tr>
+          <tr>
+            <th class="break">
+              12:00–13:00
+            </th>
+            <td>
+              Lunch
+            </td>
+          </tr>
+          <tr>
+            <th>
+              13:00–14:00
+            </th>
+            <td>
+              Breakout sessions
```

Comment:
I suggest participants to fill in the breakout grid before Day 2 lunch. Should provide a wiki or some such to propose breakout session ahead the meeting.